.nvmrc

```diff
@@ -1 +0,0 @@
-14
```

Comment:
Would it be more consistent to set it to `18`, to match the [environment in the deploy workflow](https://github.com/guardian/commercial-canaries/blob/main/.github/workflows/deploy.yaml#L19-L21)? I'm often switching versions when I change directory with `fnm`, so this would have the benefit of the version in CI matching locally automatically.

Reply:
We don't run the canaries locally so I was thinking that we don't need this file. But, we do have CDK which requires node I believe, so I'll change it to `18` .